### PR TITLE
Feature/release pipeline

### PR DESCRIPTION
**Pull request by jacobcrawford**
We describe how a helm chart is released to the helm registry we have setup in Github.

### CODE REVIEW

**Review by MartinSchmidt — DISMISSED** [review]
Looks good.

File: helm/release-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Helm release pipeline
+When a helm chart repository is being developed there is a need to continuously release these for others to use, both for new features and for bug fixes.
+
+The releases of the helm chart should be as follows: 
+ * Trigger Github workflow on merge with the main branch:
+ 1. build a helm chart package.
+ 2. creates a Github release.
+ 3. attaches the package as an asset to the release.
+ 4. push the asset to our [helm registry](https://github.com/distributed-technologies/helm-repository) and update the index file for the helm registry
```
Comment (skovbakke) [inline helm/release-pipeline.md:9]:
```suggestion
 1. package the Helm chart.
 2. create a GitHub release.
 3. attach the package as an asset to the release.
 4. push the asset to our [helm registry](https://github.com/distributed-technologies/helm-repository) and update the index file for the helm registry.
```
Made sure all the steps are worded the same way.

**Review by skovbakke — CHANGES_REQUESTED** [review]
A little wording fix

**Review by skovbakke — APPROVED** [review]
nice

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**